themes/digital.gov/layouts/partials/core/communities.html

```diff
@@ -0,0 +1,63 @@
+{{/* Digital.gov Communities Component
+  Renders a list of Digital.gov communities on both the home page and communities page
+
+  @param {string} community_title - Title of the community
+  @param {string} community_deck - Short description of the community page.
+  @param {string} community_page - Either "main" for the community page or "home" for the home page. Dictates the layout of the community cards.
+*/}}
+
+{{/* Gets all communities pages */}}
+{{ $dg_communities := (where .context.Site.Pages "Section" "communities").ByTitle }}
+
+{{/* Gather only the pages with a weight greater than or equal to 1 */}}
+{{ $dg_communities := (where $dg_communities ".Params.weight" "ge" 1) }}
+
+{{/* Gather only the pages with a dg-emphasis key */}}
+{{ $dg_communities := (where $dg_communities ".Params.dg_highlight" "eq" true) }}
+
+
+<div class="grid-container-desktop">
+  <header
+    class="page-head {{ cond (eq .community_page "home") "communities__home-title" "" }}"
+  >
+    {{- with .community_title -}}
+      <h2>{{ . | markdownify }}</h2>
+    {{- end -}}
+    {{- with .community_deck -}}
+      <p>
+        {{ . | markdownify }}
+      </p>
+    {{- end -}}
+  </header>
+</div>
+<section class="usa-section">
+  {{ if eq .community_page "main" }}
+    <div class="grid-container-desktop">
+      <div class="grid-row tablet-lg:grid-gap-2">
+        <ul class="usa-card-group">
+          {{ range $i, $e := $dg_communities }}
+            <li
+              class="usa-card tablet:grid-col-4 {{ if $e.Params.dg_acronym }}
+                cop-card--{{ $e.Params.dg_acronym }}
+              {{ end }}"
+            >
+              {{- partial "core/usa-card.html" (dict "header" $e.Params.dg_shortname "body" $e.Params.summary "link" $e.Permalink "link_title" (printf "%s%s" "Connect with <br/>" $e.Params.dg_shortname) "path" $e.Path) -}}
+            </li>
+          {{ end }}
+        </ul>
```

Comment:
Thanks! I moved the path logic outside of the partial and into this section, as it might vary depending on the context in which the partial is moved.